web-platform-config.ini

```diff
@@ -0,0 +1,21 @@
+ [Network]
+ Port = 55770
+ [Repo]
+ Logfile = ./
+ Path = ./
+ SavePeriod = 3600
+ [Extensions]
+ Directory = sc-machine/bin/extensions
+ [Stat]
+ UpdatePeriod = 1800
+ Path = /tmp/sctp_stat
+
+ ##### sc-memory
+ [memory]
+ max_loaded_segments = 1000
+
+ [kpm]
+ max_threads = 32
+
+ [debug]
+ is_debug = True
```

Comment:
There are code style problems that are described in sc-machine PR

Reply:
A separate issue has been created to tackle this: https://github.com/ostis-ai/sc-machine/issues/51
We are aware of that problem, but it's out of scope for this PR, otherwise we'd spend another month reviewing and discussing it 🤷🏻‍♂️

The goal was to decouple the modules of the platform in a non-breaking fashion rather than rewriting configuration and changing the current defaults. 
We'll do it one step at a time :)

Reply:
Okay